### DISCUSSION TITLE
Search in Milvus

### DISCUSSION THREAD

**Discussion by sampratha28** [milvus-io/milvus#31155] [Q&A and General discussion]
Does milvus sort the search results?

The reason i ask is because, with limit = 5, i am able to get context with higher score but not with limit = 3

**Comment by xiaofan-luan** [comment]
That happens, merely because of some of the search path might be ignored with smaller topk.
To solve that:
1. tune the search parameters -> if you are using HSNW, ef and efsearch need to be increased.
2. retrieve larger topk and try to rerank the result
